custom-recipes/googlesheets-append/recipe.py

```diff
@@ -30,9 +30,10 @@
 doc_id = config.get("doc_id")
 if not doc_id:
     raise ValueError("The document id is not provided")
-tab_id = config.get("tab_id")
-if not tab_id:
+tabs_ids = get_tab_ids(config)
+if not tabs_ids:
     raise ValueError("The sheet name is not provided")
+tab_id = tabs_ids[0]
```

Comment:
In this case, `tabs_ids`'s value is already a string and not a list according to logs (towards the end):
```
[2023/02/06-15:17:00.733] [null-err-43] [INFO] [dku.utils]  - 2023-02-06 15:17:00,732 INFO googlesheets plugin - config parameters: {'insert_format': 'RAW', 'preset_credentials_service_account': {}, 'oauth_credentials': {'access_token': 'HASHED_SECRET:str:214'}, 'auth_type': 'single-sign-on', 'doc_id': '1sqHyIA-X_g6N_PovWqIMyP-lhiDUKkqTi4I5MfQ2YeY', 'tab_id': '', 'tabs_ids': 'Sheet5'}
```
(because the input is a `SELECT` and not a `MULTISELECT` I guess)

As a result, `tab_id`'s value is wrong and trying to run the recipe leads to sheet not found error:
![image](https://user-images.githubusercontent.com/17353043/216996233-0a5a80d2-b6a7-4f97-849e-1b404841c770.png)
(the "S" between parentheses is actually the sheet name given to the function `get_spreadsheet` i.e. the first character of the string "Sheet5")

Reply:
I can't reproduce that on my side... What input did you have on the recipe's UI that lead to this ?

Reply:
This bug is now fixed in another branch, https://github.com/dataiku/dss-plugin-googlesheets/commit/e8cb6d98bc982df119c9df1f57bd7128caa8b884